Source/Support Files/Macaw-umbrella.h

```diff
@@ -0,0 +1,16 @@
+#ifdef __OBJC__
```

Comment:
te dwa pliki musiałeś dodać jeszcze raz? nie było ich w projekcie?

Reply:
Jak Macaw był kompilowany przy użyciu CocoaPodsów, to one tworzyły ów pliki, dzięki nim, w plikach swift-owych nie było potrzeby importowania `UIKit`.  
Natomiast jak projekt był kompilowany przez carthage, czyli bezpośrednio projekt Xcode, to do wyboru było dodanie brakujących importów, albo wstrzyknięcie właśnie tego nagłówka.